src/DropDown/dropdown-style.scss

```diff
@@ -1,94 +1,311 @@
-        >.custom-dropdown {
-            >.custom-dropdown__menu {
-                position: absolute;
-                bottom: auto;
+.custom-dropdown {
+    > .custom-dropdown-toggle {
+        background-color: white !important;
+        height: $dropdown-toggle-height !important;
+        min-width: $dropdown-width !important;
+        padding: 0px;
+        border-color: $grey-500;
+        display: flex;
+        align-items: center;
+        justify-content: space-between;
+        padding-left: 16px;
+
+        > .title {
+            font-weight: normal;
+            color: $body-color;
+        }
+
+        > .right-items {
+            display: flex;
+            > .dropdown-icon-holder {
+                display: flex;
+                align-items: center;
+                justify-content: center;
+                height: $dropdown-toggle-height;
+                width: $dropdown-toggle-height;
+
+                > svg {
+                    will-change: transform;
+                    @include transition($transition-time);
+                    width: $icon-size;
+                    height: $icon-size;
+                    fill: $blue-dark;
+                }
             }
         }
-        >.custom-dropdown:not(.custom-dropdown--is-disabled) {
-            >.custom-dropdown__control {
-                @include transition(border-color $transition-time);
-                height: 42px;
-                min-width: 200px;
-                background-color: $white;
-                cursor: pointer;
-                >.custom-dropdown__value-container {
-                    height: 42px;
-                    .custom-dropdown__input {
-                        background: $transparent;
-                    }
-                    .custom-dropdown__placeholder,
-                    .custom-dropdown__single-value {
-                        color: $black;
+
+        &.open {
+            > .right-items > .dropdown-icon-holder {
+                &.chevron {
+                    > svg {
+                        @include transform(rotate(180deg));
                     }
                 }
-                >.custom-dropdown__indicators {
-                    .custom-dropdown__indicator-separator {
-                        display: none;
-                    }
-                    .custom-dropdown__indicator {
-                        >svg {
-                            color: $blue;
-                        }
+            }
+        }
+
+        &:hover {
+            border-color: $blue;
+            background-color: white;
+        }
+
+        &:focus {
+            border-color: $blue-darker;
+            box-shadow: none;
+            background-color: white;
+            &:active {
+                color: $body-color;
+                border-color: $blue-darker;
+                box-shadow: none;
+                background-color: white;
+            }
+        }
+
+        &.more {
+            background-color: inherit !important;
```

Comment:
is it possible to not use !important ?

Reply:
resolved in commit https://github.com/sebgroup/react-components/pull/26/commits/90ff6df057917ce882a2c165a3719737dfcc0a65